opentelemetry/src/logs/record.rs

```diff
@@ -1,79 +1,46 @@
 use crate::{
-    trace::{SpanContext, SpanId, TraceContextExt, TraceFlags, TraceId},
+    trace::{SpanContext, TraceContextExt},
     Array, Key, StringValue, Value,
 };
 use std::{borrow::Cow, collections::HashMap, time::SystemTime};
 
-#[derive(Debug, Clone)]
-#[non_exhaustive]
-/// LogRecord represents all data carried by a log record, and
-/// is provided to `LogExporter`s as input.
-pub struct LogRecord {
-    /// Event name. Optional as not all the logging API support it.
-    pub event_name: Option<Cow<'static, str>>,
+/// Abstract interface for managing log records. The implementation is provided by the SDK
```

Comment:
Is the term "Abstract interface" common in Rust? 

Suggestion:
"SDK implemented trait for managing log records"

Reply:
Done.